.editorconfig

```diff
@@ -2,7 +2,7 @@
 # ** DO NOT EDIT THIS FILE **
```

Comment:
This comment is real: you can't just change this file (or pylintrc) without affecting it in the future.  These files end with a checksum that is now invalid.